### ISSUE TITLE
Downloading subtitles takes long time

### ISSUE THREAD

**Issue by Maggie-123** [exebetche/vlsub#179]
New version of VLSub (0.10.0) quickly finds the file but downloading takes loooong time, VLC offers the dialog to kill it.
Looks like finding files uses TLS 1.1 but downloading is still 1.0?
Btw, both 1.0 and 1.1 are not secure enough. I warmly suggest using 1.2.

Thank you.

**Comment by ghost** [comment]
I agree. It finds subs immediately but always get stuck on 'downloading subtitle' with the progress bar full.

I don't even think the maintainer maintains this package anymore.

**Comment by ThatLurker** [comment]
The delay might be caused by what it does after the subtitles are downloaded, like unzip, rename and move the sub file

**Comment by ghost** [comment]
Yeah probably.

**Comment by ghost** [comment]
But what if the delay is before it downloads the sub? Or before it even searches for subs?

Usually waiting for a minute would solve this but it's obnoxious.

**Comment by ThatLurker** [comment]
That might be caused by opensubtiltes servers

**Comment by ThatLurker** [comment]
Might also be caused by vlc doing other things in the background like buffering and such

**Comment by gdimi** [comment]
the problem is that VLC can't handle the zip file containing the subtitles. In my case there was a case of perl packages mismatch versions. I install latest perl and this error disappeared.
I also changed HTTP1.1 to HTTP1.0 just in case..

**Comment by ThatLurker** [comment]
Any confirmation on this working for anyone else?
